docs/collections/_schema/json-schema.md

```diff
@@ -302,6 +302,30 @@ For example, a `Network` entity may include the IP address of its gateway.
 }
 ```
 
+#### `EntityOrCommon` {#schema-entitytypes-shape-entity-or-common}
+{: .no_toc }
+
+Attributes with an entity or common type can also be referred to using an alternate `"EntityOrCommon"` syntax.
+This syntax more closely matches the Cedar schema format, and is output by the Cedar CLI when converting from the Cedar schema format to JSON.
+As an example, the following definition for the `Document` type is equivalent to the one given above in the [`Entity` section](#schema-entitytypes-shape-entity) if `User` is defined as an entity type.
+
+```json
+"Document" : {
+    "shape" : {
+        "type" : "Record",
+        "attributes" : {
+            "Owner": {
+                "type": "EntityOrCommon",
+                "name": "User"
+            }
+        }
+    }
+}
+```
+
+If `User` is defined as a common type, then `Owner` has this common type instead.
+It is an error for `User` to be defined as both an entity and common type.
```

Comment:
If `User` is defined as both an entity and common type, this resolves to preferring the common type, as specified in RFC 24.

Reply:
That's why I asked for your review 🙂 Let me know if the new text looks ok